x-pack/plugins/security_solution/public/common/hooks/use_app_toasts.test.ts

```diff
@@ -37,31 +38,36 @@ describe('useDeleteList', () => {
     expect(addErrorMock).toHaveBeenCalledWith(error, { title: 'title' });
   });
 
-  it("uses a AppError's body.message as the toastMessage", async () => {
-    const kibanaApiError = {
```

Comment:
This `kibanaApiError` was a representative error type that we intended to handle, so it's surprising to see support for it dropped here. Did we verify that this is no longer an error that we encounter/support?